.changeset/chilled-yaks-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': patch
```

Comment:
```suggestion
'@hyperlane-xyz/registry': major
```

Reply:
Whenever something is removed, that's a breaking change and the version increment will need to be `major`